tasks/copyright.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+import datetime
+import re
+import subprocess
+import sys
+from pathlib import Path, PurePosixPath
+
+GLOB_PATTERN = "**/*.go"
+
+COPYRIGHT_HEADER = f"""
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the BSD 3-Clause License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright {datetime.datetime.now().year}-present Datadog, Inc.
+// Copyright (c) 2021, RapDev.IO
+""".strip()
+
+COPYRIGHT_REGEX = [
+    r'^// Unless explicitly stated otherwise all files in this repository are licensed$',
+    r'^// under the BSD 3-Clause License\.$',
+    r'^// This product includes software developed at Datadog \(https://www\.[Dd]atadoghq\.com/\)\.$',
+    r'^// Copyright 20[1-3][0-9]-([Pp]resent|20[1-3][0-9]) Datadog, (Inc|Inmetrics)\.$',
```

Comment:
I think that's not wanted, even in the agent repo every copyright is `Datadog, Inc`
```suggestion
    r'^// Copyright 20[1-3][0-9]-([Pp]resent|20[1-3][0-9]) Datadog, Inc\.$',
```

The only matches I found are in the archived repo of the security agent 😄 
https://github.com/DataDog/datadog-security-agent/blob/f9f4adcbdec854cfa9c5981a40d2f4871ef04e7b/pkg/util/containers/providers/windows/dummy.go#L4